Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -140,11 +140,12 @@ extension SQLiteNormalizedCache: NormalizedCache {
     let result: Swift.Result<[Record?], Error>
     do {
       let records = try self.selectRecords(forKeys: keys)
+      let recordsIndexMap = records.indices.reversed().reduce(into: [:]) { resultMap, index in
+        resultMap[records[index].key, default: index] = index
```

Comment:
The `default` parameter is unused in this case. https://developer.apple.com/documentation/swift/dictionary/2894528-subscript
```suggestion
        resultMap[records[index].key] = index
```